sdk/core/azure-core-amqp/src/main/java/com/azure/core/amqp/implementation/ReactorExecutor.java

```diff
@@ -29,7 +29,7 @@
 /**
  * Schedules the proton-j reactor to continuously run work.
  */
-class ReactorExecutor implements AsyncCloseable {
+public class ReactorExecutor implements AsyncCloseable {
```

Comment:
Note: This impl type is _public_ to enable mocking via 'ReactorProvider.createExecutorForReactor' in the dependent libraries.